ngraph/test/CMakeLists.txt

```diff
@@ -635,11 +635,14 @@ if (OV_COMPILER_IS_CLANG)
     target_compile_options(unit-test PRIVATE -Wno-undef -Wno-reserved-id-macro)
 endif()
 
-if (NGRAPH_ONNX_IMPORT_ENABLE)
-    get_target_property(ONNX_IMPORTER_SRC_DIR onnx_importer SOURCE_DIR)
-    target_include_directories(unit-test PRIVATE ${ONNX_IMPORTER_SRC_DIR}/src)
+if (NGRAPH_ONNX_FRONTEND_ENABLE)
+    get_target_property(ONNX_FRONTEND_SRC_DIR onnx_ngraph_frontend SOURCE_DIR)
+    target_include_directories(unit-test PRIVATE ${ONNX_FRONTEND_SRC_DIR}/src)
 
-    target_link_libraries(unit-test PRIVATE onnx_importer)
+    target_link_libraries(unit-test PRIVATE onnx_ngraph_frontend)
+    if (LINUX)
+        target_link_options(unit-test PRIVATE -Wl,--exclude-libs,ALL)
+    endif()
```

Comment:
why do we need it? can we remove this?

Reply:
This is to not export symbols from onnx_common (in unit-test binary) so they're not bound do libonnx_ngraph_frontend.so during relocation. Since we switched protobuf from a dynamic to static library we need to hide its symbols in unit-test so libonnx_ngraph_frontend can use its own protobuf copy.